app/views/posts/index.html.erb

```diff
@@ -1 +1,37 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<h1>Here is a list of posts for a given user</h1>
+
+<div class="to_all_container">
+  <img class="user_photo" src="<%= @user.photo %>" alt="user photo">
+  <div>
+     <h2>Name: <%= @user.name %></h2>
+     <h4>Number of posts: <%= @user.post_counter %></h4>
+  </div>
+</div>
+
+
+ <% @posts.each do |post| %>
+ <div class="to_all_container">
+    <div>
+      <h2><%= post.title %></h2>
+      <p><%= post.text %></p>
+      <div>
+         <span>Comments: <%= post.comments_counter %></span>
+         <span>Likes: <%= post.likes_counter %></span>
+         <h5>
+         <%= link_to "Details", user_post_path(post.author_id, post.id), class: "btn" %>
+         </h5>
+     </div>
+    </div>  
+ </div>
+    
+  <h1>All comments</h1>
+  <div class="to_all_container">
+   <% post.comments.each do |comment| %>
```

Comment:
- [ ] The comments on this page should use the recent comments method in the post model.

- [ ] **[OPTIONAL]** When your app is started it has no root path and uses the default Rails root view as generated with the app. It would be better to define the root route so that it takes the visitor to the users' index page when they start the app. In your route file, you can solved this:

  ```
  root 'users#index'